src/Nethermind/Nethermind.Merge.AuRa/AuRaMergeConfig.cs

```diff
@@ -2,6 +2,7 @@ namespace Nethermind.Merge.AuRa
 {
     public class AuRaMergeConfig : IAuRaMergeConfig
     {
+        // No longer needed, but we can't remove it because backward compatibility. This settings is ignored
```

Comment:
What would happen if we remove it? Also AuRa Merge is not live anywhere so removing it should be doable even with compatibility issues.

Reply:
I discussed it with @jmederosalvarado, we can remove it ;)